src/ProtocolFees.sol

```diff
@@ -45,7 +46,9 @@ abstract contract ProtocolFees is IProtocolFees, Owned {
         returns (uint256 amountCollected)
     {
         if (msg.sender != protocolFeeController) InvalidCaller.selector.revertWith();
-        if (_isUnlocked()) ContractUnlocked.selector.revertWith();
+        if (!currency.isAddressZero() && CurrencyReserves.getSyncedCurrency() == currency) {
```

Comment:
Could you add a comment that even if native is synced, protocol fees cannot effect native accounting since it uses msg.value ?

Reply:
fd74a08789557068250f17756c83f75d9de4aefd